CLAPACK/Makefile.envs

```diff
@@ -1,2 +1,2 @@
-CFLAGS_OPT=-O2
-LDFLAGS_OPT=-O2
+CFLAGS_OPT=-Os -s INLINING_LIMIT=5
+LDFLAGS_OPT=-Os -s INLINING_LIMIT=5
```

Comment:
This reduces the size of the scipy package by ~4 MB / 170MB,  so it helps but not that much. Also tried with `--llvm-lto 1` and compiling all packages with `-Os` -- that didn't have much impact.